docs/dev/resources/networking.md

```diff
@@ -0,0 +1,32 @@
+# Networking Common Tasks and Information
+
+## Quick Access
+[Allow Connection to Sites on the Internet from a ProB workload](#allowing-connection-to-sites-on-the-internet)
+
+[Allow Connection to internal application](#allow-connection-to-internal-application)
+
+[DNS pitfalls](#dns-pitfalls-to-internal-application-connecting)
+
+### Allowing Connection to sites on the Internet
+This is a bit more niche, as our unclassified notebooks already have connection to the internet, but this can come up when we have a Protected B certified application like seen in https://github.com/StatCan/aaw-private/issues/125 where it's accessible to the internet, but unaccecssible from our Protected B workloads.
+
+To handle this we want to declare a [firewall_policy_rule_collection_group](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/firewall_policy_rule_collection_group) in our terraform files. The source address is the address associated with protected-b workloads. In the issue above we created it in our specific environment files as the `firewall.tf` file in our `terraform-azure-statcan-aaw-network` is mirrored and publically accessible.
+Something that isn't immediately obvious is that the `priority` should be unique, this is easier to see if you have access to the firewall resource in azure.
+
+### Allow Connection to internal application in an already peered network

```

Comment:
I think this wording is a little tricky. I'm assuming its <Allow protected-b notebook connection to cloud-main application>?

Reply:
Assuming you are talking about line 16 here about "Allow Connection to internal app.." its both classified and unclassified workloads that would be allowed to talk with the internal application.  Though you raise a good point ill add something about the source addresses